Runtime/Scripts/ExampleUDP.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using UnityEngine;
+using MobiledgeX;
+using System.Text;
+using DistributedMatchEngine;
+
+[RequireComponent(typeof(MobiledgeX.LocationService))]
+public class ExampleUDP : MonoBehaviour
+{
+    MobiledgeXIntegration mxi;
+    MobiledgeXUDPClient udpClient;
+    string udpHost; // the UDP url for your app instance
+    int udpSendPort; // the public UDP port of your app instance
+    int udpReceivePort; // You can specify the UDP port to receive messages from the server
+
+    IEnumerator Start()
+    {
+        yield return StartCoroutine(MobiledgeX.LocationService.EnsureLocation());
+        GetEdgeConnection();
+    }
+
+    async void GetEdgeConnection()
+    {
+        mxi = new MobiledgeXIntegration();
+        try
+        {
+            await mxi.RegisterAndFindCloudlet();
+        }
+        catch (DmeDnsException dde)
+        {
+            Debug.Log("Dme dns Exception: " + dde.Message);
```

Comment:
There a IsEdgeEnabled call to ensure the app has some kind of current available network status for the example?

Reply:
No